### ISSUE TITLE
facing errors while importing the file in Intel System Studio.

### ISSUE THREAD

**Issue by sirigiriram** [intel-iot-devkit/up-squared-grove-IoT-dev-kit-arduino-create#25]
i had exported cmakefile from arduino create and imported that in Intel system studio.i am getting errors while building in intel system studio.
NOTE: I have successfully compiled this program on opencv with c++. 
**System information:**
Processor: AMD,quad core
Tool: intel system studio 2018 update1
Operating System : Ubuntu 16.04
opencv 4.0
I have installed Docker also.
these are the errors i am facing. 

![isserrors](https://user-images.githubusercontent.com/38934612/39579110-97a85826-4f03-11e8-9a3d-08e9ba3ead75.png)
![isserrors2](https://user-images.githubusercontent.com/38934612/39579112-97dc5cca-4f03-11e8-8ec1-edf4161a1204.png)
![isserrors3](https://user-images.githubusercontent.com/38934612/39579115-980e31f0-4f03-11e8-93a2-9a07ad7320cb.png)

I have tried using the below path to build. please confirm, whether it is correct or not. 

![iss4](https://user-images.githubusercontent.com/38934612/39580847-bda7ad98-4f07-11e8-8660-50dd7750e05a.png)

please help me to come out of the issue

**Comment by anjgola** [comment]
We are looking into this.

**Comment by anjgola** [comment]
Can you confirm that you are using the Open CV sample from arduino create?

**Comment by sirigiriram** [comment]
no, i am using upsquared grove iot dev kit sample in arduino create.
